### ISSUE TITLE
failed to load module `zsh/pcre'

### ISSUE THREAD

**Issue by smallpvnk** [zthxxx/jovial#13]
Hi I'm trying to install this theme with oh-my-zsh on Mac OS Catalina 10.15.7. 
I get this error when I open a new terminal window after installing Jovial theme:

```
update_terminal_cwd:12: failed to load module `zsh/pcre': dlopen(/usr/lib/zsh/5.7.1/zsh/pcre.so, 9): image not found                                                                                        
update_terminal_cwd:12: -pcre-match not available for regex
regexp-replace:28: failed to load module `zsh/pcre': dlopen(/usr/lib/zsh/5.7.1/zsh/pcre.so, 9): image not found
regexp-replace:28: -pcre-match not available for regex
regexp-replace:28: failed to load module `zsh/pcre': dlopen(/usr/lib/zsh/5.7.1/zsh/pcre.so, 9): image not found
regexp-replace:28: -pcre-match not available for regex
regexp-replace:28: failed to load module `zsh/pcre': dlopen(/usr/lib/zsh/5.7.1/zsh/pcre.so, 9): image not found
regexp-replace:28: -pcre-match not available for regex
regexp-replace:28: failed to load module `zsh/pcre': dlopen(/usr/lib/zsh/5.7.1/zsh/pcre.so, 9): image not found
regexp-replace:28: -pcre-match not available for regex
```

I don't have pcre.so file in /usr/lib/zsh/5.7.1/zsh/, do you know how I would install it?

**Comment by zthxxx** [comment]
duplicate, see #12  (https://github.com/zthxxx/jovial/issues/12#issuecomment-671314357)

TL;DR

```bash
brew install zsh 
sudo chsh -s `command -v zsh` $USER
```

**Comment by zthxxx** [comment]
@smallpvnk

**Comment by smallpvnk** [comment]
Thank you that worked! Sorry I didn't see the previous question 😅
